packages/@okta/vuepress-site/docs/reference/api/system-log/index.md

```diff
@@ -294,7 +294,7 @@ The request object describes details related to the HTTP request that triggers t
 
 ### GeographicalContext Object
 
-Geographical Context describes a set of geographic coordinates. In addition to containing latitude and longitude data, this object also contains address data of postal code-level granularity. Within the [Client](#client-object) object, the geographical context refers to the physical location of the client when it sends the request that triggers this event.
+Geographical Context describes a set of geographic coordinates. In addition to containing latitude and longitude data, this object also contains address data of postal code-level granularity. Within the [Client](#client-object) object, the geographical context refers to the physical location of the client when it sends the request that triggers this event. All events with [Transaction](#transaction-object) with type equal to `WEB` should have a geograpical context set, otherwise not geographical context will be set. The geographical context data could be missing if the geographical data for a request could not be resolved.  
```

Comment:
Question about the phrase "otherwise no geographical context is set" - is this necessary? or should it be reworded to: ".....should have a geographical context set, otherwise setting the geographic context is not necessary."

Reply:
@susanharper-okta , the issue is that we are the ones setting the geographical context. maybe 'otherwise geographical context will not be set'?

Reply:
Okay, got it. I just have a couple of edits:

.....All [Transaction](#transaction-object) events with `type` equal to `WEB` will have a geographical context set. Transaction events with `type` equal to `JOB` will have no geographical context set. The geographical context data could be missing if the geographical data for a request could not be resolved.